spec/serializers/api/v3/statement_serializer_spec.rb

```diff
@@ -0,0 +1,82 @@
+# spec/serializers/api/v1/statement_serializer_spec.rb
+
+require "rails_helper"
+
+RSpec.describe API::V3::StatementSerializer, type: :serializer do
+  let(:subject) { described_class }
+  let(:cohort) { create(:cohort) }
+  let(:statement) { create(:statement, cohort:) }
+
+  it "serializes the `id`" do
+    response = subject.render_as_hash(statement)
+
+    expect(response[:id]).to eq(statement.id)
+  end
+
+  it "serializes the `cohort start year`" do
+    cohort.start_year = 2025
```

Comment:
Its better if you pass these values in `let` when you create the objects

Reply:
You can add all the other attributes, its fine to initialize it with known values so you can later test if they exist.